internal/handler_factory.go

```diff
@@ -0,0 +1,96 @@
+package internal
+
+import (
+	"time"
+
+	"github.com/wavefronthq/wavefront-sdk-go/internal/sdkmetrics"
+)
+
+type HandlerFactory struct {
```

Comment:
Could the formats be moved to this file as I think this is the only place they are used?

Reply:
I'll take a look at that - I think that they'll probably go away entirely in the next iteration of refactoring. Those values are only read in two places - once if we need to log an error, and once when we check for a special case about Events.

If we can refactor those checks out, I think the constants can go away entirely.